nmostesting/mocks/Node.py

```diff
@@ -211,6 +212,16 @@ def parse_sdp(self, sdp_data):
                 sdp_param_leg["source_ip"] = filter_line.group(2)
             else:
                 sdp_param_leg["source_ip"] = None
+
+            format_line = re.search(r"a=fmtp:(\S*\s*)(.*)", sdp_data)
+
+            if format_line and format_line.group(2):
+                #  Handle parameter keys that have no value, e.g. 'interlace' and set the value to 'True'
+                sdp_param_leg["format"] = {key: ''.join(value) if len(value) > 0 else 'True'
```

Comment:
Why string `'True'` rather than e.g. `None`, if it is just to get the `key` into the dict?

Reply:
Oh... you want to do an equality test between some typed parameters (integers, strings and Booleans at least) and the simply chopped-up strings from the `fmtp` line, so you're using `str(...)` below... Hmmm...

Reply:
The parsed SDP values are now cast to the correct type so casting to string no longer required.